app/helpers/provider_helper.rb

```diff
@@ -15,4 +15,8 @@ def google_form_url_for(settings, email, provider)
     settings.url + "&" +
       { settings.email_entry => email, settings.provider_code_entry => provider.provider_code }.to_query
   end
+
+  def is_current_cycle(cycle_year)
```

Comment:
Wondering if this logic should be in the provider decorator instead?

Reply:
Probably not worth it as we already have `Settings.current_cycle` to assert what the current cycle is. This is specific to the context of the view